.github/workflows/e2e.yaml

```diff
@@ -24,10 +24,10 @@ jobs:
         echo "${TARGET}"
         case "${TARGET}" in
           linux-amd64-e2e)
-            PASSES='build release e2e' CPU='4' EXPECT_DEBUG='true' COVER='false' RACE='true' ./scripts/test.sh
+            GOOS=linux GOARCH=amd64 CPU=4 EXPECT_DEBUG=true RACE=true make test-e2e-release
             ;;
           linux-386-e2e)
-            GOARCH=386 PASSES='build e2e' CPU='4' EXPECT_DEBUG='true' COVER='false' RACE='true' ./scripts/test.sh
+            GOOS=linux GOARCH=386 CPU=4 EXPECT_DEBUG=true RACE=true make test-e2e
```

Comment:
Why the target for `linux-386-e2e` is `test-e2e`, but it's `test-e2e-release` for `linux-amd64-e2e`? Shouldn't they be both `test-e2e`?

Reply:
Because on `linux-amd64-e2e` we also test upgrades by `PASSES='build release e2e'`. We should test upgrades on at least one target, however  maybe we can update the name to reflect that.